### PR TITLE
Fix test projects that should be ignored on linux

### PR DESCRIPTION
**Pull request by Maxwe11**
PR #1312 changes msbuild property UnsupportedOperatingSystems -> UnsupportedPlatforms so project files also should be updated.

/cc @Priya91

### CODE REVIEW

**Comment by stephentoub** [comment]
Thanks for fixing these up, @Maxwe11.  Most of these shouldn't be necessary anymore, though.  @mmitche?  The only ones that should still need them are those projects that are Windows-only, which from this list is just Microsoft.Win32.Registry.csproj, and temporarily System.Security.SecureString.csproj (we haven't done the Unix implementation yet).

**Comment by Maxwe11** [comment]
@stephentoub rebased

**Comment by mmitche** [comment]
@stephentoub The ones I originally added in there were the ones that were either Windows only or totally failing and causing the tests not to complete.  I think we should merge this change and them experiment with removing the rest.

File: src/Microsoft.Win32.Primitives/tests/Microsoft.Win32.Primitives.Tests.csproj [hunk 1]
```diff
@@ -11,7 +11,6 @@
     <AssemblyName>Microsoft.Win32.Primitives.Tests</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <FileAlignment>512</FileAlignment>
-    <UnsupportedOperatingSystems>Linux</UnsupportedOperatingSystems>
```
Comment (mmitche) [inline src/Microsoft.Win32.Primitives/tests/Microsoft.Win32.Primitives.Tests.csproj]: Why was this removed?  Doesn't this not work?

Comment (stephentoub) [inline src/Microsoft.Win32.Primitives/tests/Microsoft.Win32.Primitives.Tests.csproj]: It does work :)
